### ISSUE TITLE
Error: Unable to install this module

### ISSUE THREAD

**Issue by ashokreplies** [SAP/generator-easy-ui5#73]
Hi,

npm install -g generator-easy-ui5 command result in an error.

```
$ npm install -g generator-easy-ui5
npm ERR! code ENOTFOUND
npm ERR! syscall getaddrinfo
npm ERR! errno ENOTFOUND
npm ERR! network request to https://api.github.com/generator-easy-ui5/generator-easy-ui5/generator-easy-ui5 failed, reason: getaddrinfo ENOTFOUND api.github.com
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

```

I have tried almost everything available to install this module but failed.

Attached here the list of commands I have tried and the last log file which may be useful for your troubleshooting.

[command_prompt_cmds.txt](https://github.com/SAP/generator-easy-ui5/files/6288130/command_prompt_cmds.txt)
[2021-04-09T19_26_47_372Z-debug.log](https://github.com/SAP/generator-easy-ui5/files/6288135/2021-04-09T19_26_47_372Z-debug.log)

Thanks

**Comment by vobu** [comment]
whereever you're physically and network-wise located, there seems to be some non-standard setup involved in a network component along the way. it seems your system tries to pull the package fron github insted of npmjs.com
do you have a (gloal) .npmrc, redirecting npm install to the github package repo?
are you behind a corporate proxy?

**Comment by ashokreplies** [comment]
I did try the normal command but that failed due to certificate error. So i took the alternate routes but even that failed. 
Alternate routes i have tried are:
1. Install directly from git url with npm command
2. Install from local folder.
```
npm install -g generator-easy-ui5
npm ERR! code UNABLE_TO_GET_ISSUER_CERT_LOCALLY
npm ERR! errno UNABLE_TO_GET_ISSUER_CERT_LOCALLY
npm ERR! request to https://api.github.com/generator-easy-ui5/generator-easy-ui5/generator-easy-ui5 failed, reason: unable to get local issuer certificate

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\akkumar\AppData\Local\npm-cache\_logs\2021-04-09T21_01_58_482Z-debug.log
```

**Comment by ashokreplies** [comment]
I did try switching off the proxy setting, but it gave a different error.
![image](https://user-images.githubusercontent.com/25179438/114562331-53b68580-9c66-11eb-8efe-9307340486b8.png)

**Comment by ashokreplies** [comment]
I am going to talk the corporate network team first and if they cant help me then I will come back and let you know. 

Thank you

Kind regards
Kumar

**Comment by ashokreplies** [comment]
Found the issue with the certificate from the https://registry.npmjs.org. Because npm could not access this site due to the certificate error, it goes to the github and results in error.

I was advised to switch off the proxy during the setup which helped solve the certificate error.
Also there was another issue where npm redirecting to github as a result of a registry key.
I ran the command "npm config delete registry" to fix that problem.

I have finally installed this module. Thank you for your help. I will close this issue.
